presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/JdbcConnectorFactory.java

```diff
@@ -83,14 +88,14 @@ public Connector create(String catalogName, Map<String, String> requiredConfig,
         return injector.getInstance(JdbcConnector.class);
     }
 
+    @Deprecated
```

Comment:
add `/** @deprecated`  and tell what to use instead.

also the commit message ("Deprecate JdbcModuleProvider") could indicate  a hint as well